html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label>
```

Comment:
This `label` isn't configured correctly to match with the form control element it is paired with. The `label` and the form control element it is associated with both need a specific attribute with a matching value. The attributes are mentioned in the instructions and the documentation, please re-read them and try to find the way to properly configure a `label` element.